internal/utils/grpc.go

```diff
@@ -16,10 +19,48 @@ func CloseClientConnection(connection *grpc.ClientConn) {
 }
 
 func GrpcDialWithInsecure(serviceName string, serviceURL string) (*grpc.ClientConn, error) {
```

Comment:
Is this function even used anywhere now?

Reply:
In two `_test` files I can also replaced them there and remove this function

Reply:
Yeah, might as well replace it